app/src/main/java/com/example/perfectweatherallyear/repository/localData/WeatherDao.kt

```diff
@@ -0,0 +1,51 @@
+package com.example.perfectweatherallyear.repository.localData
+
+import android.database.sqlite.SQLiteConstraintException
+import androidx.room.*
+import com.example.perfectweatherallyear.model.DayWeather
+import com.example.perfectweatherallyear.model.HourWeather
+
+@Dao
+interface WeatherDao {
+
+    @Insert(onConflict = OnConflictStrategy.ABORT)
+    suspend fun insertDayWeather(dataWeatherData: List<DayWeather>)
```

Comment:
maybe make this fun private? Because looks like external component should use upsertForecast